docs/integrations/python.md

```diff
@@ -48,6 +48,9 @@ In case no authentication parameters exist, it is also possible to explicitly cr
 
 Here's how to instantiate a client:
 
+{: .note }
```

Comment:
This should be a section and not a note.
It should include:
1. Limitations (available only for Enterprise and Cloud only)
2. Installation instructions (with extra_requires)
3. Link to instructions on how to create the client

Reply:
@N-o-Z  Good point, and I did miss the installation instructions, but do you think this should be here or in the `external-principals-aws`? My thought was to leave it there but mention here that Enterprise customers can do this in the following link.

Reply:
Either way I think it's fine